website/static/js/fangorn.js

```diff
@@ -49,6 +49,64 @@ var getExtensionIconClass = function(name) {
     return null;
 };
 
+function findByTempID(parent, tmpID){
+    var child;
+    var item;
+    for (var i = 0; i < parent.children.length; i++) {
+        child = parent.children[i];
+        if (!child.data.tmpID) {
+            continue;
+        }
+        if (child.data.tmpID === tmpID) {
+            item = child;
+        }
+    }
+    return item;
+}
+
+function cancelUploads (row) {
+    var treebeard = this;
```

Comment:
Can we make an exception to this rule in this case? I'm using treebeard to make it clear to developers that this refers to the Treebeard API.